src/components/Header/index.jsx

```diff
@@ -1,26 +1,37 @@
 import React from 'react';
+import { Helmet } from 'react-helmet';
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
 import './Header.scss';
 
 class Header extends React.Component {
   componentDidMount() {
-    this.props.getUserInfo();
+    this.props.getSiteInfo();
   }
   render() {
+    const stylesheet = `${this.props.themeName}-app.css`;
```

Comment:
Just so I understand: So we are fetching the theme name from the server, but the actual theme file will be part of the build of the frontend app?

Reply:
Correct, theme name will be part of the SiteBranding on the server, but <theme>-app.css will be exported with the rest of the files in the frontend app's prod build.